src/test/pybind/test_rbd.py

```diff
@@ -1796,6 +1799,24 @@ def test_site_name(self):
         self.rbd.mirror_site_name_set(rados, "")
         eq(rados.get_fsid(), self.rbd.mirror_site_name_get(rados))
 
+    def test_mirror_peer_bootstrap(self):
+        eq([], list(self.rbd.mirror_peer_list(ioctx)))
+
+        self.rbd.mirror_mode_set(ioctx, RBD_MIRROR_MODE_DISABLED)
+        assert_raises(InvalidArgument, self.rbd.mirror_peer_bootstrap_create,
+                      ioctx);
+
+        self.rbd.mirror_mode_set(ioctx, RBD_MIRROR_MODE_POOL)
+        token_b64 = self.rbd.mirror_peer_bootstrap_create(ioctx)
+        token = base64.b64decode(token_b64)
+        token_dict = json.loads(token)
+        eq(['fsid', 'client_id', 'key', 'mon_host'].sort(),
+            list(token_dict.keys()).sort())
```

Comment:
@dillaman I suppose it will not do what you want. `sort` does not return value. You might want use `sorted(list)` instead.